index.html

```diff
@@ -133,18 +134,18 @@ <h3 class="article__title">How can I help?</h3>
                     </a>
                   </p>
                 </div>
-              </div>
+              </article>
             </div>
           </div>
         </div>
 
-        <div class="sidebar col-lg-4">
+        <aside class="sidebar col-lg-4">
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
+          <article class="article">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-spring-fundraisers.jpg"
+              src="/images/spring-fundraisers_thumbnail.jpg"
               alt=""
```

Comment:
Make sure to always include a description in the alt for images, it's important for screen readers

Reply:
Thanks @MCompton96 I will be extra careful with alt attribute